src/alvin0319/AmongUs/AmongUs.php

```diff
@@ -125,7 +125,8 @@ public function onEnable() : void{
 			new AmongUsCommand(),
 			new AmongUsManageCommand()
 		]);
-
+		
+		$this->getLogger()->info("Successfully Loaded AmongUs");
```

Comment:
This is unnecessary logging, when I submit this plugin to poggit release, this will be first rejected reason.

Reply:
Okay